src/main/java/edu/cornell/kfs/fp/service/RecurringDisbursementVoucherDocumentService.java

```diff
@@ -35,11 +37,33 @@
 	List<RecurringDisbursementVoucherPDPStatus> findPdpStatuses(RecurringDisbursementVoucherDocument recurringDisbursementVoucherDocument);
 	
 	/**
+	 * Cancels all DVs spawned by the Recurring Disbursement Voucher that are in saved status.
+	 * @param recurringDisbursementVoucherDocument
+	 * @param cancelMessage
+	 * @return A list of DV document numbers that have been canceled.
+	 */
+	Set<String> cancelDisbursementVouchers(RecurringDisbursementVoucherDocument recurringDisbursementVoucherDocument, String cancelMessage);
+	
+	/**
+	 * Cancels all PDP payments that are in open status there were created from the passed in recurring DV. 
+	 * @param recurringDisbursementVoucherDocument
+	 * @param cancelMessage
+	 * @return List of payment groups canceled.
+	 */
+	Set<String> cancelPDPPayments(RecurringDisbursementVoucherDocument recurringDisbursementVoucherDocument, String cancelMessage);
+	
+	/**
 	 * Find all of the payment detail records associated with a recurring DV.
 	 * @param recurringDisbursementVoucherDocument
 	 * @return
 	 */
 	Collection<PaymentDetail> findPaymentDetailsFromRecurringDisbursementVoucher(RecurringDisbursementVoucherDocument recurringDisbursementVoucherDocument);
 	
-	
+	/**
+	 * Determins if an individual payment can be canceled from a recurring DV.
+	 * @param user
+	 * @param paymentDetailStatus
+	 * @return
```

Comment:
Please clean up the Javadocs for this method, especially the misspelling in the description.